src/transfer.ts

```diff
@@ -50,16 +50,17 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2):
   const currentSpendingBalance = microAlgos(balance - minimumBalanceRequirement.microAlgos)
 
   if (minSpendingBalance > currentSpendingBalance) {
+    const from = fundingSource ?? (await getDispenserAccount(algod))
```

Comment:
Should we propagate an optional kmd instance to here so it's possible to control the kmd instance that's used? All of the other things that use kmd have that pattern.